README.md

```diff
@@ -23,28 +23,65 @@ Review](https://badges.ropensci.org/688_status.svg)](https://github.com/ropensci
 
 With `distionary`, you can:
 
-1.  [Specify](./articles/specify.html) a probability distribution, and
+1.  Specify a probability distribution
+    ([built-in](./articles/specify-built-in.html) or [your
+    own](./articles/specify-user-defined.html)), and
 2.  [Evaluate](./articles/evaluate.html) the probability distribution.
 
 The main purpose of `distionary` is to implement a distribution object,
 and to make distribution calculations available even if they are not
-specified in the distribution. `distionary` powers the wider
-[`probaverse` ecosystem](https://probaverse.com) for making probability
-distributions that are representative of your data, not just
-out-of-the-bag distributions like the Normal, Poisson, etc.
-
-`distionary` makes reference to common terms regarding probability
-distributions. If you’re uneasy with these terms and concepts, most
-intro books in probability will be a good resource to learn from. As
-`distionary` develops, more documentation will be made available so that
-it’s more self-contained.
+specified in the distribution’s definition. `distionary` provides the
+building blocks of the wider [`probaverse`
+ecosystem](https://probaverse.com) for making representative statistical
+models.
 
 The name “distionary” is a portmanteau of “distribution” and
 “dictionary”. While a dictionary lists and defines words, `distionary`
 defines distributions and makes a list of common distribution families
 available. The built-in distributions act as building blocks for the
 wider probaverse.
 
+## Statement of Need
```

Comment:
[<img src="https://badges.ws/badge/comment-6272a4" alt="comment"/>](https://github.com/katrinabrock/katrinabrock/blob/main/review-comment-badges.md)

Might change the heading to something less formal like "Why Distionary?"